src/platform/telink/telink-mbedtls-config.h

```diff
@@ -0,0 +1,29 @@
+/******************************************************************************
+ * Copyright (c) 2022 Telink Semiconductor (Shanghai) Co., Ltd. ("TELINK")
```

Comment:
Does this need CHIP license attribution?

Reply:
Thx, fixed. Also I will fix the same in develop branch in next Telink PR.